src/main/java/org/cactoos/text/Synced.java

```diff
@@ -48,13 +47,14 @@ public Synced(final Text text) {
      * @param lck The lock
      */
     public Synced(final Text text, final Object lck) {
-        super(new Scalar<String>() {
-            @Override
-            public String value() throws Exception {
-                synchronized (lck) {
-                    return text.asString();
+        super(
+            new TextOf(
+                () -> {
```

Comment:
@victornoel Can we reused `scalar.Synched` here?

Reply:
@andreoss good catch